File: Codility/PermCheck.py
def solution(A):
    # write your code in Python 3.6
    if max(A) != len(A) or len(set(A)) != len(A):
        return 0
    return 1


A = [4, 1, 3]
B = [4, 1, 3, 2]

print(solution(A), solution(B))

File: Codility/PermCheck/PermCheck.js
function solution(A) {
  // write your code in JavaScript (Node.js 8.9.4)
  const maxVal = Math.max.apply(null, A);
  const set = new Set(A);
  if (maxVal !== A.length || set.size !== A.length) {
    return 0;
  }
  return 1;
}

const A = [4, 1, 3];
const B = [4, 1, 3, 2];

console.log(solution(A), solution(B));
